### ISSUE TITLE
Text based diagrams are missing in the documentation

### ISSUE THREAD

**Issue by jmini** [docToolchain/docToolchain#666]
**Describe the bug**
Some images in the Documentation website are missing.

**To Reproduce**
Go to the website:
https://doctoolchain.github.io/docToolchain/#_text_based_diagrams

> Section "3.2.1. Text based Diagrams"

**Expected behavior**
You should see the image.

**Current output**

The `src` attribute of the generated image is absolute in the CI runner:

```html
<div class="imageblock">
    <div class="content">
        <img src="/home/travis/build/docToolchain/docToolchain/build/docs/images/plantUMLdemoPlantUML.png"
            alt="plantUMLdemoPlantUML" width="246" height="236">
    </div>
    <div class="title">Figure 2. example diagram</div>
</div>
```

Something might go wrong during the publication of the docs.

**Screenshots**
<img width="979" alt="Screenshot" src="https://user-images.githubusercontent.com/1222165/137801368-3c9dbacb-3d5e-4b41-b5b2-af99f795b976.png">

**Additional context**
This is also reported in the htmlChecks report:
https://doctoolchain.github.io/docToolchain/htmlchecks/index.html

> * image "/home/travis/build/docToolchain/docToolchain/build/docs/images/plantUMLdemoPlantUML.png" missing
> * image "/home/travis/build/docToolchain/docToolchain/build/docs/images/plantUMLdemoPlantUMLWidth.png" missing
> * image "/home/travis/build/docToolchain/docToolchain/build/docs/images/plantUMLdemoPlantUMLScale.png" missing
 
You also see the error in the dev tools:

```
plantUMLdemoPlantUML.png:1 Failed to load resource: the server responded with a status of 404 ()
plantUMLdemoPlantUMLWidth.png:1 Failed to load resource: the server responded with a status of 404 ()
plantUMLdemoPlantUMLScale.png:1 Failed to load resource: the server responded with a status of 404 ()
```

**Comment by jmini** [comment]
Maybe this will be fixed when you use the `v2.0.x` output at the root of the published documentation:

https://doctoolchain.org/docToolchain/v2.0.x/015_tasks/03_task_generateHTML.html#_text_based_diagrams

On this page the image is present

**Comment by rdmueller** [comment]
thanx for the PR! I guess we can close this now...
